client/index.html

```diff
@@ -7,6 +7,7 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0, maximum-scale=1"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100..900&display=swap" rel="stylesheet">
```

Comment:
What's the rationale for adding this new font download?

Reply:
We just thought it would look better for the warning text. Should we take it out?

Reply:
Seems like a significant dependency to add solely for the text of an info message that is not used frequently. Let's see what it looks like with the default without it.